### PR TITLE
Small fix in orchestrating-third-party-services-with-oauth2.adoc

### PR DESCRIPTION
**Pull request by wmedvede**
**JIRA:**

**Description:**

Please make sure that your PR meets the following requirements:

- [X] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [X] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [X] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [X] The nav.adoc file has a link to this guide in the proper category
- [X] The index.adoc file has a card to this guide in the proper category, with a meaningful description

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @hmanwani-rh would you mind review this small fix?
Thanks!

File: serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc [hunk 1]
```diff
@@ -615,7 +618,10 @@ Example response::
 In the previous examples, the currencies supported by the `currency-exchange-workflow` include `EUR`, `USD`, `JPY`, `GBP`, `CAD`, `BRL`, and `AUD`. However, the `acme-financial-service` REST service can resolve any type of currency exchange. This is an example of a Serverless Workflow implementing the intermediate data filtering, transforming, and validations.
 ====
 
-.Example of unexpected errors when accessing `acme-financial-service`
+**Example 3)** Unexpected errors when accessing `acme-financial-service`
+
+To execute this case, first you must go the terminal window where you started the `acme-financial-service` and stop it with CTRL+C.
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:623]:
```suggestion
To execute the previous case, you must go to the terminal window where you started the `acme-financial-service` and stop the service using `CTRL+C`.
```

File: serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc [hunk 2]
```diff
@@ -571,7 +573,8 @@ Example response::
 ----
 ====
 
-.Example of unsuccessful execution when calculating the currency exchange from `EUR` to `MXN`
+**Example 2)** Unsupported currency error when calculating the currency exchange from `EUR` to `MXN`
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:576]:
```suggestion
The following is an example of an unsupported currency error when calculating the currency exchange from `EUR` to `MXN`:
```

File: serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc [hunk 3]
```diff
@@ -531,7 +531,9 @@ java -jar target/quarkus-app/quarkus-run.jar
 . When all the services are running, use the following `curl` commands to run the `currency-exchange-workflow`:
 +
 --
-.Example of successful execution when calculating the currency exchange from `EUR` to `USD`:
+
+**Example 1)** Successful execution when calculating the currency exchange from `EUR` to `USD`
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:535]:
```suggestion
The following is an example of successful execution when calculating the currency exchange from `EUR` to `USD`:
```

**Review by hmanwani-rh — COMMENTED** [review]
@wmedvede I've added some suggestions below. Please take a look and let me know if you've any questions.

**Comment by wmedvede** [comment]
@hmanwani-rh your suggestions where added.
Thanks!

File: serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc [hunk 4]
```diff
@@ -615,7 +617,10 @@ Example response::
 In the previous examples, the currencies supported by the `currency-exchange-workflow` include `EUR`, `USD`, `JPY`, `GBP`, `CAD`, `BRL`, and `AUD`. However, the `acme-financial-service` REST service can resolve any type of currency exchange. This is an example of a Serverless Workflow implementing the intermediate data filtering, transforming, and validations.
 ====
 
-.Example of unexpected errors when accessing `acme-financial-service`
+The following is an example of the occurrence of unexpected errors when accessing `acme-financial-service`:
+
+To execute, you must go to the terminal window where you started the `acme-financial-service` and stop the service using `CTRL+C`.
+
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:623]:
```suggestion
Before you launch the command, you must go to the terminal window where you started the `acme-financial-service` and stop the service using `CTRL+C`.

```

**Review by radtriste — APPROVED** [review]

**Comment by ricardozanini** [comment]
Please, once it's done cherry-pick it to 1.24.x

**Review by hmanwani-rh — APPROVED** [review]

**Pull request merged**